test/cat.jl

```diff
@@ -397,8 +454,8 @@ end
     @test err.value.msg == "column(s) B are missing from argument(s) 1"
     # multiple missing 1 column
     err = @test_throws ArgumentError vcat(df1, df2, df2, df2, df2, df2)
-    err2 = @test_throws ArgumentError reduce(vcat, [df1, df2, df2, df2, df2, df2])
-    @test err == err2
+    @test err.value.msg == "column(s) B are missing from argument(s) 2, 3, 4, 5 and 6"
+    err = @test_throws ArgumentError reduce(vcat, [df1, df2, df2, df2, df2, df2])
     @test err.value.msg == "column(s) B are missing from argument(s) 2, 3, 4, 5 and 6"
```

Comment:
Is this really the error that is thrown? I would expect something about only two arguments, and if it was identical to the previous error then `@test err = err2` would be enough.

Reply:
It's the error that is thrown. it's to check that `reduce(vcat, dfs)` behaves exactly similar to `vcat(dfs...)` 

However the `reduce(vcat,...)` error probably shouldn't be that since it refers to `argument(s)` and the `reduce` version has only 2 arguments. But I don't really want special error handling for just a small optimization of `recuce(vcat, dfs)`. 

I changed the code to test that the error messages are equal for now.

Reply:
OK, sounds fine. It's not a problem if the error is smarter than what we expect. ;-)